checks/E215A6.yaml

```diff
@@ -0,0 +1,62 @@
+id: "E215A6"
+name: SAPHanaFilesystem resource configured
+group: Pacemaker
+description: |
+  SAPHanaFilesystem resource is configured
+remediation: |
+  ## Abstract
+  SAPHanaFilesystem resource should be configured especially for hana_scale_out clusters. So, that problems with HANA filesystem can be monitored and remediative steps can be taken by cluster if the resource is hung due to the fileystem unavailability.
```

Comment:
Something looks wrong with the text after the first dot. For example, we are missing `the` in the `by cluster`.